test/components-pages/fixtures/old-hydrate-no-create-no-input/tests.js

```diff
@@ -0,0 +1,25 @@
+var path = require("path");
+var expect = require("chai").expect;
+
+describe(path.basename(__dirname), function() {
+    it("should mount, not call the onCreate/onInput methods and update", function() {
```

Comment:
Where did you check that the methods wasn't called?

Reply:
He added throws in the lifecycle (https://github.com/marko-js/marko/pull/1289/files#diff-809b6711d16268823b5b2c50c4366ed4R4) which would cause the test to fail.